bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/internal/AudioManagerImpl.java

```diff
@@ -206,7 +206,7 @@ public AudioSink getSink() {
         String regex = pattern.replace("?", ".?").replace("*", ".*?");
         Set<String> matchedSources = new HashSet<String>();
 
-        for (String aSource : audioSinks.keySet()) {
+        for (String aSource : audioSources.keySet()) {
```

Comment:
Made a PR with this change: https://github.com/eclipse/smarthome/pull/3119. Ignored the tests, related to removing the change from this branch.